README.md

```diff
@@ -1,14 +1,21 @@
-# purescript-colors
+# Colors
 
-[![Latest release](http://img.shields.io/bower/v/purescript-colors.svg)](https://github.com/sharkdp/purescript-colors/releases)
-[![Build Status](https://api.travis-ci.org/sharkdp/purescript-colors.svg?branch=master)](https://travis-ci.org/sharkdp/purescript-colors)
+[![CI](https://github.com/purescript-contrib/purescript-colors/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-colors/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](https://img.shields.io/github/release/purescript-contrib/purescript-colors.svg)](https://github.com/purescript-contrib/purescript-colors/releases)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-colors/badge)](https://pursuit.purescript.org/packages/purescript-colors)
+[![Maintainer: sharkdp](https://img.shields.io/badge/maintainer-sharkdp-teal.svg)](https://github.com/sharkdp)
```

Comment:
I believe @sharkdp would prefer someone else be the primary maintainer for the library. If no one is currently taking over then we can remove this line.